### PR TITLE
Update README of onnx2circom

### PR DESCRIPTION
**Pull request by exfinen**
This is to add additional information that is not obvious and is helpful to people who are new to mpcstats.

### CODE REVIEW

File: tests/onnx2circom/README.md [hunk 1]
```diff
@@ -39,34 +56,41 @@ git clone https://github.com/mhchia/MP-SPDZ
 cd MP-SPDZ
 git remote add kevin_mpc https://github.com/mhchia/MP-SPDZ.git
```
Comment (mhchia) [inline tests/onnx2circom/README.md:57]:
```suggestion
```

File: tests/onnx2circom/README.md [hunk 2]
```diff
@@ -39,34 +56,41 @@ git clone https://github.com/mhchia/MP-SPDZ
...
 git fetch kevin_mpc
```
Comment (mhchia) [inline tests/onnx2circom/README.md:58]:
```suggestion
```

File: tests/onnx2circom/README.md [hunk 3]
```diff
@@ -39,34 +56,41 @@ git clone https://github.com/mhchia/MP-SPDZ
...
-git checkout arith-executor
+git checkout -b arith-executor kevin_mpc/arith-executor
```
Comment (mhchia) [inline tests/onnx2circom/README.md:59]:
```suggestion
git checkout arith-executor
```

**Review by mhchia — APPROVED** [review]
Looks great! About MP-SPDZ, wow about we just use the remote `origin` and remove the parts related to `kevin_mpc`?

**Comment by exfinen** [comment]
That have made the procedure simpler and I reflected the changes. Also I add a note for Ubuntu 22.04. Can you take a look again?

**Comment by mhchia** [comment]
Looks good, thanks!

**Pull request merged**